lib/tmuxinator/project.rb

```diff
@@ -135,17 +131,14 @@ def pre_window
       elsif pre_tab?
```

Comment:
I just noticed that there are several conditional branches in this method. While multiple commands don't make sense for `rbenv` or `rvm`, would you mind also adding a call to `parsed_parameters` here with `yaml["pre_tab"]`?